packages/ses/src/intrinsics.js

```diff
@@ -17,6 +21,8 @@ import {
   whitelist,
 } from './whitelist.js';
 
+const isFunction = obj => typeof obj === 'function';
```

Comment:
This participates in an incidental refactor below, avoiding an unnecessary closure allocation for filtering functions.